docs/eventing/broker/broker-event-delivery.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Event delivery"
+weight: 03
+type: "docs"
+showlandingtoc: "false"
+---
+
+You can configure how events are delivered for each broker by adding a `delivery` spec, as shown in the following example:
+
+```yaml
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  name: with-dead-letter-sink
+spec:
+  delivery:
+    deadLetterSink:
+      ref:
+        apiVersion: serving.knative.dev/v1
+        kind: Service
+        name: example-sink
+    backoffDelay: <duration>
+    backoffPolicy: <policy-type>
+    retry: <integer>
+```
+
+Where
+
+- The `deadLetterSink` spec contains configuration settings to enable using a dead letter sink. This tells the subscription what happens to events that cannot be delivered to the subscriber. When this is configured, events that fail to be delivered are sent to the dead letter sink destination. The destination can be a Knative service or a URI. In the example, the destination is a `Service` object, or Knative service, named `example-sink`.
```

Comment:
The sink can also be a Kubernetes Service, if that's useful to mention. Technically any Addressable that conforms to the Eventing Sink contract will work here, but that's probably too much detail.